tests/verifiers/multiplier_verifier.fe

```diff
@@ -0,0 +1,206 @@
+use std::buf::{MemoryBuffer, MemoryBufferWriter, MemoryBufferReader}
```

Comment:
please create a project structure and test the actual modules

Reply:
There is no way to test the actual groth16.fe module since the actual module for the specific circom circuit is generated after snarkjs. Instead, I named the folder "examples" for the specific multiplier_verifier.fe file